docs/release-notes.asciidoc

```diff
@@ -7,6 +7,29 @@
 :issue: https://github.com/elastic/kibana/issues/
 :pull: https://github.com/elastic/kibana/pull/
 
+[discrete]
+[[release-notes-7.13.2]]
+== 7.13.2
+
+[discrete]
+[[known-issue-7.13.2]]
+==== Known issue
+The following ML rules contain incorrectly configured ML job IDs (underscores were used instead of dashes between words) and cannot be successfully activated after they are enabled. Running these rules will cause an error message to display, indicating that an error occurred during the rule's execution. This issue is present in 7.13, 7.13.1, and 7.13.2.
+
+* `high-count-by-destination-country`
+* `high-count-network-denies`
+* `high-count-network-events`
+* `rare-destination-country`
+
+To ensure these rules can sucessfully run, duplicate the rule and edit it using these steps:
+
+. Go to the Detections page and select **Manage detection rules**.
+. Filter the Rules table to only display rules with the `ML` tag and search for the ML rule you want to duplicate.
+. Select the rule you want to duplicate and click **Bulk actions -> Duplicate selected**.
+. Select the duplicated rule and click **Edit rule settings**.
+. From the Definition tab, enter the correct ML Job ID. For example, to fix the incorrectly configured `high_count_by_destination_country` ML rule job ID, remove the current job ID and enter `high-count-by-destination-country`. Click **Save changes** after you've finished.
+. Delete the prebuilt ML job.
```

Comment:
Is it worth linking out to the Kibana issue (https://github.com/elastic/kibana/issues/102146) like some of the previous Known Issues?

Reply:
Yes, can absolutely do that. Appreciate the reminder! :)